File: C/beginner/1010.c
/*Including Libraries*/
#include <stdio.h>

/*Main Function*/
int main(){
    /*Variable declaration and initialization*/
    int piece_code1, piece_code2, quantity1, quantity2;
    double price1, price2, total_value;

    /*Requesting values to user*/
    scanf("%d" "%d" "%lf", &piece_code1, &quantity1, &price1);
    scanf("%d" "%d" "%lf", &piece_code2, &quantity2, &price2);
    
    /*Doing the calculations*/
    total_value = (quantity1 * price1) + (quantity2 * price2);

    /*Showing results. For some reason, beecrowd doesn't accept my solution with for. Sad... :( */
    printf("VALOR A PAGAR: R$ %.2lf\n", total_value);

    /*Ending Algorithm*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/

File: Haskell/beginner/1010.hs
-- Importing functions
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables (Ok, this is harder than I expected. Thanks again Google!)
    order_1 <- getLine
    order_2 <- getLine

    -- Extracting values:
    let [_, amnt1, value1] = map read (words order_1) :: [Double] -- 1
    let [_, amnt2, value2] = map read (words order_2) :: [Double] -- 2

    -- Doing the calculations
    let total = (amnt1 * value1) + (amnt2 * value2)

    -- Showing results
    printf "VALOR A PAGAR: R$ %.2f\n" total

-- By Fernando Serra
-- https://github.com/fernandosserra